app/src/main/res/layout/item_follow_request.xml

```diff
@@ -1,77 +1,102 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="72dp"
+    android:layout_height="wrap_content"
     android:gravity="center_vertical"
     android:paddingLeft="16dp"
     android:paddingRight="16dp">
 
-    <ImageView
-        android:id="@+id/avatar"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:layout_alignParentStart="true"
-        android:layout_centerVertical="true"
-        android:layout_marginEnd="24dp"
-        android:contentDescription="@string/action_view_profile" />
-
-    <LinearLayout
+    <androidx.emoji.widget.EmojiTextView
+        android:id="@+id/notificationTextView"
         android:layout_width="wrap_content"
-        android:layout_height="match_parent"
-        android:layout_toEndOf="@id/avatar"
-        android:layout_toStartOf="@id/acceptButton"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:drawableStart="@drawable/ic_person_add_24dp"
+        android:drawablePadding="10dp"
+        android:ellipsize="end"
         android:gravity="center_vertical"
-        android:orientation="vertical">
+        android:maxLines="1"
+        android:paddingStart="28dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:textColor="?android:textColorTertiary"
+        android:textSize="?attr/status_text_medium"
+        tools:text="Someone requested to follow you"
+        />
 
-        <androidx.emoji.widget.EmojiTextView
-            android:id="@+id/displayNameTextView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:maxLines="1"
-            android:textColor="?android:textColorPrimary"
-            android:textSize="?attr/status_text_large"
-            android:textStyle="normal|bold"
-            tools:text="Display name" />
+    <RelativeLayout
```

Comment:
Please don't nest a `RelativeLayout` and a `LinearLayout` inside a `ConstraintLayout`. You can place all views directly inside the `ConstraintLayout`.